easybuild/framework/easyblock.py

```diff
@@ -2434,6 +2436,9 @@ def run_step(self, step, step_methods):
         """
         self.log.info("Starting %s step", step)
         self.update_config_template_run_step()
+
```

Comment:
I'd rather have the logic for dealing with not having any hooks defined in `run_hook` rather than having to copy-paste the logic to check whether any hooks are defined in multiple places.

Reply:
Seems reasonable.